update-minor-versions.mjs

```diff
@@ -0,0 +1,35 @@
+import { execSync } from "child_process";
+import fs from "fs";
+
+function readJson(filepath) {
+  return JSON.parse(fs.readFileSync(filepath, "utf-8"));
+}
+
+function bumpStudioPlugin() {
+  console.log("... bumping studio-plugin");
+  execSync("npm version minor -w=packages/studio-plugin");
+  const packageJson = readJson("./packages/studio-plugin/package.json");
+  const newVersion = packageJson?.version;
+  if (!newVersion) {
+    throw new Error("Could not parse studio-plugin version from package.json");
+  }
+  return newVersion;
+}
+
+function bumpStudio(newVersion) {
+  console.log(`... bumping studio to ${newVersion}`);
```

Comment:
nit: if we're not enforcing that the versions stay in sync, then I think we should remove `newVersion` from this log because it could be misleading

Reply:
oh I just meant changing it to `console.log("... bumping studio")` or something like we have for bumping studio-plugin. I think it's still nice to have it log something, even though we don't know what the version will be